src/404.html

```diff
@@ -0,0 +1,8 @@
+---
+layout: layouts/default
+title: Oops! Not Found
+permalink: 404.html
+---
+<h1>{{ title }}</h1>
+<p>The page you were looking for does not exist yet. Go back to <a href="{{ '/' | url }}">home page?</a></p>
```

Comment:
@jhung do you have suggestions for the 404 page both in terms of title and content. Do we want something that looks interesting or just something basic?

Reply:
@BlueSlug could you too weigh in on this?

Reply:
I'd wait to see what @jhung says, though I would suggest something along the lines of: "The page you're looking for does not exist. Here are some helpful starting points: ...." (list the home page, News, etc.)

You could also try a friendlier message like "We can't seem to find the content you're looking for", or a cheeky one like "This isn't the content you're looking for". I think aiming for clarity is going to be the best bet, regardless, and anything is better than what's there now, which is a default Nginx error page.

Reply:
Something basic is fine. We'll probably end up revisiting this with a redesign anyway.

I would change the title to "Page not found" instead of "Oops! Not Found".

Reply:
Updated.